debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -168,7 +168,7 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
 
             final PostgresEventMetadataProvider metadataProvider = new PostgresEventMetadataProvider();
 
-            final PostgresEventDispatcher<TableId> dispatcher = new PostgresEventDispatcher<>(
+            final PostgresEventDispatcher<TableId> dispatcher = new PostgresEventDispatcher<TableId>(
```

Comment:
```suggestion
            final PostgresEventDispatcher<TableId> dispatcher = new PostgresEventDispatcher<>(
```
I actually feel this change is a bit superfluous.  

The main problem with this line of code is the fact that `CommonConnectorConfig` does not type-parameterize the returned `TopicNamingStrategy` so that it avoids this unchecked warning entirely.  If that is fixed, there is no need to explicitly specify the `TableId` as done here and even doing so only moves the problem from the dispatcher instantiation to a warning on the constructor argument itself.

I'd simply leave this as-is and we fix the warning by adjusting the method signatures in `CommonConnectorConfig` instead.

Reply:
@Naros ok, removed